src/frontend/komponenter/Fagsak/Høyremeny/høyremeny.less

```diff
@@ -1,11 +0,0 @@
-.høyremeny {
-    display: flex;
-    flex-direction: column;
-    width: 25rem;
-    height: calc(100vh - 8rem);
-
-    .hendelsesbeskrivelse {
```

Comment:
Jeg fant ingen bruk for `.hendelsesbeskrivelse`, derfor fjernet jeg stylingen helt. Er det noe jeg overser her må dere rope ut 😄